components/helpers/new-guid.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Returns a new GUID
+ */
+define('new-guid', [], () =>
+    function newGuid () {
+        return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+
+            const r = Math.random() * 16 | 0;
+            // eslint-disable-next-line no-mixed-operators
+            const v = (c === 'x') ? r : (r & 0x3 | 0x8);
```

Comment:
Clarify the expression with parentheses to avoid the need to disable the linting rule.